sdk/storage/azure-storage-blob/azure/storage/blob/_shared/base_client.py

```diff
@@ -78,9 +78,10 @@ def __init__(
         self._hosts = kwargs.get("_hosts")
         self.scheme = parsed_url.scheme
 
-        if service not in ["blob", "queue", "file"]:
+        if service not in ["blob", "queue", "file-share"]:
```

Comment:
wow! we also renamed the endpoint?

Reply:
Haha it's not rename  I saw you parsed it in the next line

Reply:
Haha - yes we did this because the new package name is used to pass into the pipeline policies on line 105 :)

Reply:
ooh!! Thanks for this info. Do you know why we need to pass package name when we create a pipeline?
`self._create_pipeline(self.credential, storage_sdk=service, **kwargs)`